src/component/mxgraph/shape/event-shapes.ts

```diff
@@ -157,6 +157,7 @@ export class StartEventShape extends EventShape {
 export class EndEventShape extends EventShape {
   public constructor(bounds: mxgraph.mxRectangle, fill: string, stroke: string, strokewidth: number = StyleConstant.STROKE_WIDTH_THICK) {
     super(bounds, fill, stroke, strokewidth);
+    this.isUsingThrowIcons = true;
```

Comment:
As this variable is use not only in Intermediate Event, I propose to rename it:
```suggestion
    this.withFilledIcon = true;
```

Reply:
I chose the `throw` naming because even for `non intermediate event`, there is the concept of `catch` and `throw event`.
For instance
- `message start event` is triggered when a message is caught/received by the event and then starts the process
- `signal end event` throws/sends a signal to another BPMN elements

That being said, from a shape point of view, we can use the `fill` wording as we only care about how we paint/draw the icon :stuck_out_tongue_winking_eye:
So feel free to commit the proposed suggestion.

Reply:
No problem with me. 
I proposed that, because in the BPMN specification, they talk about _filled/unfilled_  icon for the different events :wink:

Reply:
so, if the BPMN spec is using this wording, let do the change